src/anyio/abc/_streams.py

```diff
@@ -209,3 +209,42 @@ async def serve(
         :param task_group: the task group that will be used to start tasks for handling
             each accepted connection (if omitted, an ad-hoc task group will be created)
         """
+
+
+class ObjectStreamConnectable(Generic[T_co]):
```

Comment:
should this set the metaclass to abc.ABC?

Reply:
No, as `Generic` already uses `ABCMeta` as its metaclass.

Reply:
No, I was wrong about that. I'll add it here too.